app/helpers/application_helper/toolbar/miq_ae_tools_simulate_center.rb

```diff
@@ -6,6 +6,7 @@ class ApplicationHelper::Toolbar::MiqAeToolsSimulateCenter < ApplicationHelper::
       N_('Copy object details for use in a Button'),
       N_('Copy'),
       :url       => "resolve",
-      :url_parms => "?button=copy"),
+      :url_parms => "?button=copy",
+      :klass     => ApplicationHelper::Button::ButtonWithoutRbacCheck),
```

Comment:
@dclarizio OK, I've found the button:

![screenshot from 2017-02-20 16-11-28](https://cloud.githubusercontent.com/assets/1187051/23130583/4fe339d6-f787-11e6-9993-da49fb61d716.png)
 
It's not a toolbar button though